test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -635,7 +634,7 @@ def symbolic_custom_invalid_add(g, input, other, alpha=None):
         f = io.BytesIO()
 
         try:
-            with self.assertRaises(torch.onnx.errors.CheckerError):
+            with self.assertRaises(torch.onnx.errors.OnnxExporterError):
```

Comment:
Why is this change?

Reply:
I think it's related to the custom op registration, where the regex was actually denying "aten" namespace. Let's relax that.

Reply:
CheckerError is actually what we want